projects/aca-playwright-shared/src/utils/utils.ts

```diff
@@ -46,4 +48,43 @@ export class Utils {
   static formatDate(date: string): string {
     return new Date(date).toLocaleDateString('en-US');
   }
+
+  /** 
+   * Method used to login user with navigation. Also waits for the page to load after login
+   * 
+   * @param loginPage page context passed from the test
+   * @param username username string
+   * @param password password string
+   * @param errorMessage error message string if the login fails
+   * 
+  */
+  static async tryLoginUser(loginPage: LoginPage, username: string, password: string, errorMessage = 'Error: '): Promise<void> {
```

Comment:
ah sorry, I did not notice you have ':' in console.error so now you will have duplicated ':' :D

Can you remove just that character ':' from errorMessage? :)